.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: cpp
+env:
+  global:
+  - FORCE_INSTALL=1
+
+matrix:
+  include:
+ - os: linux
```

Comment:
You have a mistake here. Should be two spaces. I have already correct the same issue in OpenCorePkg:
https://github.com/acidanthera/OpenCorePkg/commit/1d68511a6a835b8b4005d90d0d16e0f0d899f496

Reply:
Will fix